src/View/View.js

```diff
@@ -226,20 +228,30 @@ export default function View({
                   defaultWidth="fill"
                   footer={
                     <PaneFooter
-                      renderEnd={<Button buttonStyle="primary" data-testid="submit-applications-modal" onClick={() => onSave(checkedIdsMap, onClose)}><FormattedMessage id="stripes-core.button.saveAndClose" /></Button>}
-                      renderStart={<div style={{ alignText: 'right', display:'block' }}><FormattedMessage id="ui-plugin-select-application.totalSelected" values={{ count: Object.keys(checkedIdsMap).length }} /></div>}
+                      renderEnd={
+                        <Button aria-label={intl.formatMessage({ id: 'stripes-core.button.saveAndClose' })} buttonStyle="primary" data-testid="submit-applications-modal" onClick={() => onSave(checkedIdsMap, onClose)}>
+                          <FormattedMessage id="stripes-core.button.saveAndClose" />
+                        </Button>
+                      }
+                      renderStart={
+                        <div style={{ alignText: 'right', display:'block' }}>
+                          <FormattedMessage id="ui-plugin-select-application.totalSelected" values={{ count: Object.keys(checkedIdsMap).length }} />
+                        </div>
+                      }
                     />
                   }
                   padContent={false}
                   renderHeader={
                     () => <PaneHeader
                       firstMenu={renderResultsFirstMenu(activeFilters)}
+                      id={<FormattedMessage id="ui-plugin-select-application.applications" />}
                       paneSub={renderResultsPaneSubtitle}
                       paneTitle={<FormattedMessage id="ui-plugin-select-application.applications" />}
                     />
                   }
                 >
                   <MultiColumnList
+                    ariaLabel={intl.formatMessage({ id:'ui-plugin-select-application.listOfApplications' })}
```

Comment:
This translation key does not exist in `translations/ui-plugin-select-application/en.json`. Maybe you added it but forgot to include the file?

Reply:
`aria-label` wasn't needed here, so this is resolved now.